src/containers/blocks.jsx

```diff
@@ -155,7 +165,6 @@ class Blocks extends React.Component {
                 this.setLocale();
             } else {
                 this.props.vm.refreshWorkspace();
-                this.updateToolbox();
```

Comment:
One small bug that I'm noticing is that if I'm in the blocks tab and click on a category (e.g. Variables), switch to the costumes tab, resize the browser window (e.g. make it smaller), and switch back to the blocks tab, the variables category is still selected but the toolbox is scrolled back up to the motion category.

This seems relatively small and can probably be filed as an issue for later.

Reply:
@kchadha that seems to be a pre-existing issue, i can repro that on current production.

Reply:
@paulkaplan the behavior is not the same on production. On production, the toolbox scrolls to the top and the selected category also switches to 'Motion', with these changes, the category stays as 'Variables' even though you're seeing 
'Motion' blocks